JSDemos/Demos/TreeView/TreeViewWithSearchBar/description.md

```diff
@@ -1 +1,11 @@
-This demo demonstrates the TreeView component&rsquo;s searching capabilities. The component can find nodes containing your search string or&nbsp;only starting with it&nbsp;depending on&nbsp;the search mode.
\ No newline at end of file
+To add a search bar to the TreeView and customize the search functionality, do the following:
+
+- Set the [searchEnabled](/Documentation/ApiReference/UI_Components/dxTreeView/Configuration/#searchEnabled) property to **true**.
+
+- Use the [searchMode](/Documentation/ApiReference/UI_Components/dxTreeView/Configuration/#searchMode) property to specify whether items should contain (default), start with, or match the search string. In this example, you can change the **searchMode** in the drop-down menu.
+
+- Configure the [searchExpr](/Documentation/ApiReference/UI_Components/dxTreeView/Configuration/#searchExpr) property to specify custom search data fields. The default search field is [text](/Documentation/ApiReference/UI_Components/dxTreeView/Configuration/items/#text). 
```

Comment:
Can I use more than one data field to search against simultaneously? Did you use the "custom" word only to contrapose this scenario to "the default search field" which is `text`?

I would have said that the TreeView uses the `text` property to determine the data field for search. Then, I would have said that you can use the `searchExpr` property to specify the fields against which I should allow users to search. That is only if I have understood the relationship between the mentioned properties correctly.

BTW, can I use the `text` property if I want to use a certain dataset field as a search source?